fetch.bs

```diff
@@ -4773,6 +4977,9 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
     `<code>Transfer-Encoding</code>`/`<code>chunked</code>` and <var>response</var> is transferred
     via HTTP/1.0 or older, then return a <a>network error</a>.
 
+    <li><p>Set <var>timingInfo</var>'s <a for="fetch timing info">response start time</a> to
```

Comment:
That doesn't match how response start is [defined today](https://www.w3.org/TR/resource-timing-2/#dom-performanceresourcetiming-responsestart) (e.g. for 100 responses)

Reply:
I'm not sure this is the right place for the response, as it's not mentioned anywhere that we're waiting for the response to start arriving and that it has started arriving here. @annevk - thoughts?

Reply:
I'll put it a bit earlier, with the description copied from RT

Reply:
It seems the current text leaves response start time undefined for responses from the cache, responses from `blob:` URLs, etc. Is that intended?